lib/mayaUsd/render/vp2RenderDelegate/basisCurves.cpp

```diff
@@ -1130,11 +1130,22 @@ void HdVP2BasisCurves::_UpdateDrawItem(
                 // Assign with the index to the dormant wireframe color by default.
                 colorIndices.resize(instanceCount, 0);
 
+                // The active & lead selection information tells us the scene delegate instance
+                // Id. We need to convert that to the Vp2 instance id used in the colorIndices
+                // vector.
+                VtIntArray delegateToVp2Map = static_cast<HdVP2Instancer*>(instancer)
+                                                  ->ComputeInstanceIdToVp2InstanceIdArray(id);
+
```

Comment:
It is very similar to the mesh version. The only difference is I don't have to track is any instances are visible here. In the basis curve case I know some instances are visible because of the earlier check, and we re-use a single render item to draw the dormant curves as well as the lead & active items. The only thing we have to get right is which items do we change the color of, and that is what the new changes do.